packages/web/src/components/modal-radio/ModalRadioItem.tsx

```diff
@@ -84,7 +74,7 @@ export const ModalRadioItem = (props: ModalRadioItemProps) => {
           {tag ? <Tag className={styles.tag}>{tag}</Tag> : null}
         </Tooltip>
       </div>
-      {hintContent ? <Hint icon={hintIcon}>{hintContent}</Hint> : null}
+      {hint ? <Box pt='s'>{hint}</Box> : null}
```

Comment:
do we want a text around this when hint is a string?

Reply:
or a <Hint> even

Reply:
i updated this so hint is a reactnode, so should be chillin

Reply:
sry my point is that the arg takes string or ReactNode, so in the case it's just a string, we might want to apply harmony text styling to it

If we're never passing string, that's fine but could remove it from the prop type